### DISCUSSION TITLE
"smoothChunkBuilding"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#62] [Ideas]
While you toggle the Chunk Borders, you are not able to break / place blocks out from your standing chunk.
Pretty helpful with high speed breaking but you still would like to cut edges smoothly.

![image](https://github.com/Minecraft-AMS/Carpet-AMS-Addition/assets/120551916/24ac1eea-a6a5-4098-9345-8df0a5b8b037)

**Comment by wendavid552** [comment]
So that means you would like to limit your breaking and placing in a certain field? I think it should a client-side feature, which cannot be supported by our mod.
By the way you can try breaking the edge first? or you can try Tweakeroo, you can use it to limit your breaking.

**Reply by ghost** [reply]
Was you ever trying it yourself? It's easy to write but not to play 😂

I had specific the server support in mind - maybe some code references from the Spawn Chunks protection feature.

Not sure what you mean with limit breaking. I think this mod does not the wished behaviour of respecting the chunk borders. Individual break times for blocks are not helpful in this case, but thanks 😊

**Reply by wendavid552** [reply]
Well, it is of the breaking restrictions feature in Tweakeroo.  This feature is more like to be in a client-side mod.

Most time when digging a perimeter we always dig the edges first (The edges are usually irregular), so we hardly faces with this problem 😂.

**Reply by ghost** [reply]
Thanks, I will look for a different solution then.
